sample.py

```diff
@@ -1,10 +1,253 @@
-"""Sample code to test the checker."""
+# This is a test file for the allowed checker.
 
+# Unit 2
+# =, name, constant, def, return, function call, import, attribute,
+# +, -, *, /, //, %, **, -x, help, min, max, round, print,
+# math: floor, ceil, trunc, pi
+
+# Constant
+CONSTANT = 42
+
+# Name
+name = "example"
+
+# Function definition, return, and function call
+def square(x):
+    return x ** 2
+
+result = square(CONSTANT)
+
+# Import and attribute
+import math
+pi = math.pi
+
+# Arithmetic operations
+addition = 1 + 2
+subtraction = 5 - 3
```

Comment:
While one construct per line is good for error reporting, it makes this file long and thus harder to check what's included in the file.

Reply:
I wonder if it's best to leave sample.py as-is for the moment and create a separate all.py file with this. Probably the Python project already has files to test their ast module.

Reply:
Yeah, that's fair enough. It seems like the closest thing to what we want in the test_file for ast is contained in the [exec_tests](https://github.com/python/cpython/blob/7703def37e4fa7d25c3d23756de8f527daa4e165/Lib/test/test_ast.py#L36) and [eval_tests](https://github.com/python/cpython/blob/7703def37e4fa7d25c3d23756de8f527daa4e165/Lib/test/test_ast.py#L194) lists. I believe they have all the constructs, but since `allowed` checks for specific names of constructs It might not help that much?

Reply:
I am thinking now it might be better to just close this pull request. I probably got the wrong idea about what was needed, and I am still not  sure exactly what is required (although looking back at #8 I must have missed the part where you said concise :smile:) maybe #8 needs updating or closing aswell?